### ISSUE TITLE
Create Asynchronous implementation for DB connection type

### ISSUE THREAD

**Issue by nuzil** [magento/async-import#19]
### Summary
Currently system requires to have AMQP connection configured in case you want to use Asynchronous/Bulk API.
For make system more flexible we need functionality to make default magento DB queue works as Async.

For allow DB connection type we have to create new Published and Consumer.
Also on a step of generating *.xml files (topics, publishers) we have to detect IF AMQP is available, then use it, otherwise use DB connection and generate data for this connection

Implementation hints:

```
\Magento\WebapiAsync\Code\Generator\Config\RemoteServiceReader\Publisher::read
Magento/WebapiAsync/etc/queue_consumer.xml
Magento/WebapiAsync/etc/queue_topology.xml
```

Consider adding MassConsumer and MassPublisher to MysqlQueue

**Comment by nuzil** [comment]
Hi @Stepa4man can you give some update related to this issue?

**Comment by nuzil** [comment]
Hey @Stepa4man 
Any updates here?

**Comment by Stepa4man** [comment]
Hey @nuzil 
Not Yet, but I'd like to. I'll have time next week. I hope I'll make something.

**Comment by jimmy-lion** [comment]
Hi, any updates on this?

thanks

**Comment by naydav** [comment]
Closed because this repository is related only Asynchronous Import Functionality

@nuzil, if the task is still actual, please create an issue in
https://github.com/magento/community-features for a feature request
or in https://github.com/magento/magento2/issues for a bug
Thanks

**Comment by Winfle** [comment]
I think, this issue is important, since this module is included in 2.3 and requires an amqp brocker installation, which is not required (optional) technology Magento can use.
That causes warning messages in system logs, which forces people install RabbitMQ or disable related modules, which is not always possible in different situations.
Please, correct me if I'm wrong. Thank you
